tests/unit/client.py

```diff
@@ -766,3 +771,31 @@ def test_extension_list(self):
                     "hil.ext.switches.mock",
                     "hil.ext.switches.nexus",
                 ]
+
+
+class TestShowNetworkingAction:
+    """Test calls to show networking action method"""
+
+    def test_get_status(self):
+        """(successful) call to get_status"""
+        response = C.node.connect_network(
+                'node-01', 'eth0', 'net-01', 'vlan/native'
+                )
+        status_id = response['status_id']
+
+        response = C.node.show_networking_action(status_id)
+        assert response['status'] == 'PENDING'
+        assert response['node'] == 'node-01'
+        assert response['nic'] == 'eth0'
+        assert response['new_network'] == 'net-01'
+        assert response['channel'] == 'vlan/native'
+        assert response['type'] == 'modify_port'
```

Comment:
Any reason not to just have one `assert response == {...}`?

Reply:
oh yeah, i just forgot to do it here.